archunit/src/test/java/com/tngtech/archunit/core/domain/DependencyTest.java

```diff
@@ -68,7 +104,7 @@ public void Dependency_from_throws_declaration() {
                 .get(ClassWithDependencyOnThrowable.class).getMethod("method");
         ThrowsDeclaration<JavaMethod> throwsDeclaration = getOnlyElement(origin.getThrowsClause());
 
-        Dependency dependency = Dependency.tryCreateFromThrowsDeclaration(throwsDeclaration).get();
+        Dependency dependency = Dependency.tryCreateFromThrowsDeclaration(throwsDeclaration).iterator().next();
```

Comment:
I'd probably use _`getOnlyElement`_ from `com.google.common.collect.Iterables`, wihch also ensures that there is only one element.